perception/traffic_light_fine_detector/README.md

```diff
@@ -0,0 +1,65 @@
+# traffic_light_fine_detector
+
+## Purpose
+
+It is a package for traffic light detection using YoloX-s.
+
+## Training Information
+
+### Pretrained Model
+
+The model is based on [YOLOX](https://github.com/Megvii-BaseDetection/YOLOX) and the pretrained model could be downloaded from [here](https://github.com/Megvii-BaseDetection/YOLOX/releases/download/0.1.1rc0/yolox_s.pth).
+
+### Training Data
+
+The model was fine-tuned on around 17,000 TIER IV internal images of Japanese traffic lights.
+
+### Trained Onnx model
+
+- <https://drive.google.com/uc?id=1USFDPRH9JrVdGoqt27qHjRgittwc0kcO>
+
+## Inner-workings / Algorithms
+
+Based on the camera image and the global ROI array detected by `map_based_detection` node, a CNN-based detection method enables highly accurate traffic light detection.
+
+## Inputs / Outputs
+
+### Input
+
+| Name            | Type                                                       | Description                                      |
+| --------------- | ---------------------------------------------------------- | ------------------------------------------------ |
+| `~/input/image` | `sensor_msgs/Image`                                        | The full size camera image                       |
+| `~/input/rois`  | `autoware_auto_perception_msgs::msg::TrafficLightRoiArray` | The array of ROIs detected by map_based_detector |
+
```

Comment:
[Q]
No need to add `~/expect/rois `as input?

Reply:
fixed in commit d00c441f8dbdf8ffcfe3d53bb640d6416cbfac16

Reply:
Thanks!